Control/test/lib/mocha-status-service.js

```diff
@@ -115,34 +115,34 @@ describe('StatusService test suite', () => {
   });
 
   describe('Test Grafana Status', async () => {
-    const config = {grafana: {port: 8082}, http: {hostname: 'localh'}};
-    const expectedInfo = {hostname: 'localh', port: 8082};
+    const config = {grafana: {url: 'http://localhost:8084'}};
+    const expectedInfo = {protocol: 'http:', hostname: 'localhost', port: 8084};
 
     it('should successfully retrieve status and info about Grafana that it is running', async () => {
       const status = new StatusService(config, {}, {});
       const grafanaStatus = await status.getGrafanaStatus();
-      nock(`http://${config.http.hostname}:${config.grafana.port}`)
+      nock(config.grafana.url)
         .get('/api/health')
         .reply(200, {});
-      expectedInfo.status = {ok: true, configured: true};
-      assert.deepStrictEqual(grafanaStatus, expectedInfo);
+      assert.deepStrictEqual(grafanaStatus.status, {ok: true, configured: true});
+      assert.equal(grafanaStatus.protocol, expectedInfo.protocol);
```

Comment:
`equal` is deprecated for a while now.
It is better to use `deepStrictEqual` for objects and `strictEqual` for primitive types